documentation/Overview.asciidoc

```diff
@@ -118,7 +118,7 @@ The Vaadin documentation is arranged in the following sections and we recommend
 * <<polymer-templates/tutorial-template-model-encoders#,PolymerTemplate, Using Model Encoders>>
 * <<polymer-templates/tutorial-template-list-bindings#,PolymerTemplate, Creating Contents Dynamically Based on Items List>>
 * <<polymer-templates/tutorial-template-parent-layout#,PolymerTemplate, Using Parent Layout>>
-* <<polymer-templates/tutorial-template-mapped-components-limitations#,PolymerTemplate, Mapped Components Limitations>>
+* <<polymer-templates/tutorial-template-mapped-components-limitations#,Mapped Components Limitations>>
```

Comment:
Since this applies now to Lit, it could probably be moved up above the sections that start with `PolymerTemplate, ` as this is now considered "legacy".